Project.toml

```diff
@@ -1,20 +1,25 @@
 name = "NLopt"
 uuid = "76087f3c-5699-56af-9a33-bf431cd00edd"
-version = "0.6.6"
+version = "0.7.0"
```

Comment:
Might as well bump this to 1.0, since it hasn't changed in a while, and version numbers ≥ 1.0 provide strictly more information in semver.

Reply:
```suggestion
version = "1.0"
```

Reply:
@stevengj done!